fuel-core/src/schema/tx.rs

```diff
@@ -324,6 +324,8 @@ impl TxMutation {
         // for read-only calls.
         utxo_validation: Option<bool>,
     ) -> async_graphql::Result<Vec<receipt::Receipt>> {
+        todo!("Migrate dry-run to the new trigger architecture");
```

Comment:
Since this method only returns receipts, I don't think it needs to interact with the orchestrator since this doesn't impact regular block production.

I can make a PR to add this functionality.